src/app/Main.java

```diff
@@ -18,7 +18,12 @@ public static void main(String[] args) {
     // various cards, and the layout, and stitch them together.
 
     // The main application window.
-    JFrame application = new JFrame("Login Example");
+    JFrame application = new JFrame("MSGR");
+    // Load main window icon
+    ImageIcon icon = new ImageIcon("assets/images/MSGR_Icon.png");
+    Image image = icon.getImage();
```

Comment:
Any idea why we create an ImageIcon and then call getImage instead of doing something like

`ImageIO.read(new File("assets/images/MSGR_Icon.png"))`

Reply:
The ImageIcon class just continues execution as normal if we don't find the icon.
The icon is not critical to our application; therefore, this is the desired behaviour.